File: src/main/java/com/kopyl/commit/ErrorNotification.java
package com.kopyl.commit;

import com.intellij.notification.Notification;
import com.intellij.notification.NotificationType;
import com.intellij.notification.Notifications;

public class ErrorNotification {
    public static void show(String errorMessage) {
        Notification notification = new Notification("Error Notifications", "Auto commit message", errorMessage, NotificationType.ERROR);
        Notifications.Bus.notify(notification);
    }
}

File: src/main/kotlin/com/kopyl/commit/ErrorNotification.kt
package com.kopyl.commit

import com.intellij.notification.Notification
import com.intellij.notification.NotificationType
import com.intellij.notification.Notifications

object ErrorNotification {
    fun show(errorMessage: String) {
        val notification = Notification("Error Notifications", "Auto commit message", errorMessage, NotificationType.ERROR)
        Notifications.Bus.notify(notification)
    }
}
